data/json/items/armor/torso_clothes.json

```diff
@@ -1009,7 +1009,7 @@
       },
       {
         "id": "swordocracy_tshirt",
-        "name": { "str": "Down with Swordocracy t-shirt", "str_pl": "Down with Swordocracy t-shirts" },
+        "name": { "str": "Down with Swordocracy t-shirt" },
```

Comment:
https://github.com/CleverRaven/Cataclysm-DDA/blob/5c39685badae210e526cfbf572d0364c0e100a5b/src/translation.cpp#L133

Plural form checker finds `with` in the item name and thinks it is a compound phrase, so manually giving a plural form is suggested.

Reply:
Thanks! I saw the error, but wasn't sure where it comes from, should be fixed now, thanks for explaining! :)